coordinator/src/main/scala/filodb.coordinator/queryplanner/HighAvailabilityPlanner.scala

```diff
@@ -360,6 +384,7 @@ class HighAvailabilityPlanner(dsRef: DatasetRef,
       buddyGrpcEndpoint = Some(remoteGrpcEndpoint.get)
     )
     val context = qContext.copy(plannerParams = haPlannerParams)
+    logger.info(context.getQueryLogLine("Using shard level failover"))
```

Comment:
It would be useful to have lists of shards such as:
```
Materializing locally: [0, 1, 2, 3]
Materializing on buddy: [4, 5, 6, 7]
```
I know technically that info exists in the plan structure, but this would be a nice quick reference to have.